### ISSUE TITLE
Complied Version fails 

### ISSUE THREAD

**Issue by ntha314159** [TomRoush/PdfBox-Android#146]
I intend to use the library for rendering pdf files to be used for printing on thermal receipt printers. The code compiles, but crashes during runtime . The details below 

```
PDFRenderer renderer = new PDFRenderer(document);
// Render the image to an RGB Bitmap
try {
  //pageImage = renderer.renderImage(iPageNo, 1, Bitmap.Config.RGB_565);
     pageImage = renderer.renderImageWithDPI(1,203, Bitmap.Config.RGB_565);
}
catch (IOException e) {
     e.printStackTrace();
}
```

Run time crash
```
Caused by: java.lang.NullPointerException: Attempt to get length of null array
 at android.graphics.DashPathEffect.<init>(DashPathEffect.java:35)
 at com.tom_roush.pdfbox.rendering.PageDrawer.setStroke(PageDrawer.java:566)
 at com.tom_roush.pdfbox.rendering.PageDrawer.strokePath(PageDrawer.java:575)
 at com.tom_roush.pdfbox.contentstream.operator.graphics.StrokePath.process(StrokePath.java:20)
 at com.tom_roush.pdfbox.contentstream.PDFStreamEngine.processOperator(PDFStreamEngine.java:798)
 at com.tom_roush.pdfbox.contentstream.PDFStreamEngine.processStreamOperators(PDFStreamEngine.java:460)
 at com.tom_roush.pdfbox.contentstream.PDFStreamEngine.processStream(PDFStreamEngine.java:437)
 at com.tom_roush.pdfbox.contentstream.PDFStreamEngine.processPage(PDFStreamEngine.java:146)
 at com.tom_roush.pdfbox.rendering.PageDrawer.drawPage(PageDrawer.java:153)
 at com.tom_roush.pdfbox.rendering.PDFRenderer.renderPage(PDFRenderer.java:151)
 at com.tom_roush.pdfbox.rendering.PDFRenderer.renderImage(PDFRenderer.java:113)
 at com.tom_roush.pdfbox.rendering.PDFRenderer.renderImageWithDPI(PDFRenderer.java:69)
```
Any help is highly appreciated.

**Comment by TomRoush** [comment]
This should be fixed in v1.8.10.0

**Comment by Bridouille** [comment]
Hi, I'm having the exact same issue with version `1.8.10`, is there a new release that fix this issue?

**Comment by TomRoush** [comment]
Fixed in release v1.8.10.1
